### PR TITLE
Update `Using Boot` section in README.md

### PR DESCRIPTION
**Pull request by whysoserious**
Add a link to boot-clj wiki.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,16 +190,18 @@ A minimal `profiles.clj` for CIDER would be:
...
+all of their projects using a `~/.profile.boot` file like so:
 
 ```clojure
 (require 'boot.repl)
 
-(swap! boot.repl/*default-dependencies*
-       concat '[[cider/cider-nrepl "0.9.0"]])
+(swap! boot.repl/*default-dependencies* conj
+       '[cider/cider-nrepl "0.9.0"]
+       '[refactor-nrepl "1.1.0-SNAPSHOT"])
```
Comment (bbatsov) [inline README.md]: Please, remove the line about `refactor-nrepl` as it's not really a part of cider and will likely confuse someone.

File: README.md [hunk 2]
```diff
@@ -190,16 +190,18 @@ A minimal `profiles.clj` for CIDER would be:
...
 
-(swap! boot.repl/*default-middleware*
-       conj 'cider.nrepl/cider-middleware)
+(swap! boot.repl/*default-middleware* conj
+       'cider.nrepl/cider-middleware
+       'refactor-nrepl.middleware/wrap-refactor)
```
Comment (bbatsov) [inline README.md]: Same here.

**Comment by bbatsov** [comment]
I think you should submit the `refactor-nrepl` stuff to its own README. This is outside the scope of CIDER itself.

File: README.md [hunk 3]
```diff
@@ -190,16 +190,18 @@ A minimal `profiles.clj` for CIDER would be:
 ### Using Boot
 
 Boot users can configure the tool to include the middleware automatically in
-all of their projects using a `~/.boot/profile.boot` file like so:
+all of their projects using a `~/.profile.boot` file like so:
```
Comment (expez) [inline README.md]: Didn't we recently have PR making the opposite change?  Authors using different versions of boot?

Comment (bbatsov) [inline README.md]: Great point. I was wondering about this myself...

**Pull request merged**